courses/free-courses-en.md

```diff
@@ -1548,6 +1548,7 @@
 
 ### R
 
+* [CS50's Introduction to Programming with R](https://learning.edx.org/course/course-v1:HarvardX+CS50R+R/home) - Carter Zenke
```

Comment:
I think https://cs50.harvard.edu/sql/2024/ is a better page to link to.

Reply:
https://cs50.harvard.edu/r/2024/ It is the link which must be linked @eshellman

Reply:
https://cs50.harvard.edu/r/2024/ .It will be better page link for CS50 R . @eshellman the page link you suggested is for SQL course